assets/test/identity-different/Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version:5.4
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "identity-different",
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "identity-different",
+            targets: ["identity-different"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        .package(name: "cmark", url: "https://github.com/apple/swift-cmark.git", .branch("main")),
```

Comment:
```suggestion
        .package(name: "cmark", url: "https://github.com/apple/swift-cmark.git", .branch("gfm")),
```

The `main` branch is exactly a copy of `cmark`.